src/pages/sighting/featuredPhoto/FeaturedPhoto.jsx

```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react';
```

Comment:
Now that we're using this component in multiple areas of the app we should move it to `src/components`

Reply:
Done.